modules/Components/src/client/AbstractFormController.coffee

```diff
@@ -161,6 +161,7 @@ class AbstractThingFormController extends AbstractFormController
 				showDescription: field.fieldSettings.showDescription
 				editablePicklist: field.fieldSettings.editablePicklist
 				editablePicklistRoles: field.fieldSettings.editablePicklistRoles
+				parameter: field.fieldSettings.parameter
```

Comment:
(Cont.) This also part of the fix to the issue where the selected parameter in the PickList would display its id along with the string (ie. "solubilityc178" instead of "solubility")